tests/utils.py

```diff
@@ -390,7 +469,7 @@ def module_from_template(
     config: configparser.ConfigParser = configparser.ConfigParser()
     config.read(config_file)
     if new_version is not None:
-        config["metadata"]["version"] = str(new_version)
+        config["metadata"]["version"] = new_version.base_version
```

Comment:
Why use `base_version` here?

Reply:
I believe this is the convention for Python packages. At least it is what we use for our other repos: `metadata.version` contains the base version and `egg_info.tag_build` any additional tags.

Reply:
True. I confused the V1 and the V2 metadata file.